src/main/scala/is/hail/annotations/Annotation.scala

```diff
@@ -182,4 +182,84 @@ object Annotation {
       case _ => a
     }
   }
+
+  def safeFromRegionValue(t: Type, rv: RegionValue): Annotation =
+    safeFromRegionValue(t, rv.region, rv.offset)
+
+  def safeFromRegionValue(t: Type, region: Region, offset: Long): Annotation = t match {
```

Comment:
I think this can just be`Annotation.copy(t, UnsafeRow.read(t, region, offset))`.